src/components/ArticleStrip/ArticleStrip.stories.mdx

```diff
@@ -67,3 +67,19 @@ A summary of a Quartz article, typically used when linking to that article in a
 		{args => <ArticleStrip {...args} />}
 	</Story>
 </Canvas>
+
+### Sponsored article (a.k.a. bulletin)
```

Comment:
this is confusing, but technically this is "sponsor content" because it is written by Quartz Creative. we reserve "sponsored" for editorial content that has been sponsored by an advertiser.

```suggestion
### Sponsor content (a.k.a. bulletin)
```

Reply:
ah good to know